### PR TITLE
Dropdown dynamic direction

### PR DESCRIPTION
**Pull request by Advaith3600**
The dropdown box will dynamically choose which direction to upon into (up or down) by comparing its height against the height of the parent.

image

### CODE REVIEW

**Comment by arukompas** [comment]
Hey @Advaith3600 

Just had time to test this. I see this weird clipping now when hovering on the dropdown button:

Screenshot 2022-09-04 at 08 37 07

Other than that, just fix the merge conflict and this looks great! 💪 Really good fix, thanks buddy

**Review by arukompas — CHANGES_REQUESTED** [review]
See the above comment https://github.com/opcodesio/log-viewer/pull/66#issuecomment-1236264375

**Comment by Advaith3600** [comment]
@arukompas I have fixed the clipping issue and rebased the PR

**Review by arukompas — APPROVED** [review]

File: resources/views/livewire/file-list.blade.php [hunk 1]
```diff
@@ -37,11 +40,12 @@
                 <div
                     x-ref="panel"
                     x-show="open"
-                    x-transition.origin.top.right
+                    x-transition
```
[the remark this answers is not in the document]
Comment (arukompas) [reply resources/views/livewire/file-list.blade.php:43]: Hm, I'll think about it. But for now, the PR looks good, so merging! 🚢

**Comment by arukompas** [comment]
Thanks for your help @Advaith3600 today I've learned about pointer events none 😉

File: resources/views/livewire/file-list.blade.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (arukompas) [reply resources/views/livewire/file-list.blade.php:43]: @Advaith3600 just FYI, this is what I came up with for this problem. Move the control of the transition origin from HTML/JS to CSS instead:
https://github.com/opcodesio/log-viewer/pull/82

**Pull request merged**